SlideServer.py

```diff
@@ -157,4 +157,42 @@ def getSlide(image_name):
         return flask.send_from_directory(app.config["UPLOAD_FOLDER"], filename=image_name, as_attachment=True)
     else:
         return flask.Response(json.dumps({"error": "File does not exist"}), status=404)  
-            
+
+
+
+class upload():
+    uploadedStatus = "False"
+uploadStatus = upload();
```

Comment:
Why is this outside of the function; wouldn't this mean that it would report the upload to be complete for subsequent incomplete uploads, provided the first finished?

Reply:
I think you're right. 
This is silly of me. I'm on it.

Reply:
Looks like sharing variables among route functions in flask is not reliable at all. 
I am working on something which checks for the file size on the url and matches on the disk if it's downloaded completely or not.